test/src/utils.ts

```diff
@@ -28,11 +28,6 @@ import {
 } from '../../lib/notebook/notebook/modelfactory';
 
 import {
-  LatexRenderer, PDFRenderer, JavascriptRenderer,
-  SVGRenderer, MarkdownRenderer, TextRenderer, HTMLRenderer, ImageRenderer
-} from '../../lib/renderers';
```

Comment:
We still need to import HTMLRenderer and TextRenderer for the custom renderers defined in this file.